docs/source/release_notes/release-notes.rst

```diff
@@ -42,23 +44,71 @@ develop
+            +===================================+===========================+
+            | :strike:`enableSweep`             | **enabled**               |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepPauseMillis`        | **pauseMillis**           |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepReadLimit`          | **readLimit**             |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepCandidateBatchHint` | **candidateBatchHint**    |
+            +-----------------------------------+---------------------------+
+            | :strike:`sweepDeleteBatchHint`    | **deleteBatchHint**       |
+            +-----------------------------------+---------------------------+
+
+           Specifying any of the above install options will result in **AtlasDB failing to start**.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1976>`__)
+
     *    - |userbreak|
          - The Atomix algorithm implementation for the TimeLock server and the corresponding configurations have been removed.
            The default algorithm for ``TimeLockServer`` has been changed to Paxos.
            This should not affect users as Atomix should not have been used due to known bugs in the implementation.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2082>`__)
 
-    *    - |improved|
-         - The dropwizard independent implementation of the TimeLock server has been separated into a new project, ``timelock-impl``.
-           This should not affect users directly, unless they depended on classes from within the TimeLock server.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2081>`__)
-
     *    - |userbreak|
          - The previously deprecated RocksDBKVS has been removed.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1966>`__)
 
+    *    - |fixed| |improved|
+         - Sweep now dynamically adjusts the number of blocks, i.e., (cell, ts) pairs, across runs:
+
+           - On a failure run, sweep halves the number of blocks to read and to delete on subsequent runs.
+           - On a success run, sweep slowly increases the number of blocks to read and to delete on subsequent runs, up to a configurable maximum.
+
+           This should fix the issue where we were unable to sweep cells with high number of mutations.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2060>`__)
+
     *    - |new|
-         - AtlasDB now instruments embedded time and lock services, even if no leader block is present in the config,
+         - AtlasDB now instruments embedded timestamp and lock services, even if no leader block is present in the config,
            to expose aggregate response time and service call metrics.
```

Comment:
Let's include a note that this may cause a minor performance hit, and if it does we can address it by toggling the system property accordingly